packaging/amazon-linux/amazon-ecr-credential-helper.spec

```diff
@@ -0,0 +1,126 @@
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/credentials/stscreds))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/csm))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/defaults))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/ec2metadata))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/endpoints))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/request))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/session))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/aws/signer/v4))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/internal/ini))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/internal/sdkio))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/internal/sdkrand))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/internal/sdkuri))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/internal/shareddefaults))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/json/jsonutil))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/jsonrpc))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/query))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/query/queryutil))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/rest))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/service/ecr))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/service/ecr/ecriface))
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/service/sts))
+Provides:       bundled(golang(github.com/davecgh/go-spew/spew))
+Provides:       bundled(golang(github.com/docker/docker-credential-helpers/credentials))
+Provides:       bundled(golang(github.com/golang/mock/gomock))
+Provides:       bundled(golang(github.com/jmespath/go-jmespath))
+Provides:       bundled(golang(github.com/konsorten/go-windows-terminal-sequences))
+Provides:       bundled(golang(github.com/mitchellh/go-homedir))
+Provides:       bundled(golang(github.com/pkg/errors))
+Provides:       bundled(golang(github.com/pmezard/go-difflib/difflib))
+Provides:       bundled(golang(github.com/sirupsen/logrus))
+Provides:       bundled(golang(github.com/stretchr/testify/assert))
+Provides:       bundled(golang(golang.org/x/crypto/ssh/terminal))
+Provides:       bundled(golang(golang.org/x/net/context))
+Provides:       bundled(golang(golang.org/x/sys/unix))
+Provides:       bundled(golang(golang.org/x/sys/windows))
+
+%description
+The Amazon ECR Docker Credential Helper is a credential helper for the Docker
+daemon that makes it easier to use Amazon Elastic Container Registry.
+
+%prep
+%setup -c
+
+%build
```

Comment:
One last update: the `%build` section can be simplified a bit:

```rpm-spec
%build
export GOPATH="$(pwd)/_gopath"
mkdir -p "_gopath/src/github.com/awslabs"
ln -sv "$(pwd)" "_gopath/src/github.com/awslabs/amazon-ecr-credential-helper"
cd "_gopath/src/github.com/awslabs/amazon-ecr-credential-helper"
make
```

Reply:
Done